scripts/pre_commit_linter.py

```diff
@@ -598,17 +598,19 @@
 _MESSAGE_TYPE_SUCCESS = 'SUCCESS'
 _MESSAGE_TYPE_FAILED = 'FAILED'
 _TARGET_STDOUT = python_utils.string_io()
+_STDOUTS = multiprocessing.Manager().list()
```

Comment:
Use `_STDOUT_LIST` instead, it's more descriptive

Reply:
Done